app/src/main/java/org/gdg/frisbee/android/arrow/ArrowActivity.java

```diff
@@ -311,12 +312,16 @@ public void showNoNfc() {
         Toast.makeText(this, R.string.no_nfc_use_qr_scanner, Toast.LENGTH_LONG).show();
     }
 
-
+    @Nullable
     private String getEncryptedMessage() throws Exception {
```

Comment:
This code below isn't enough (line 376)?

                if (TextUtils.isEmpty(encryptedMessage)) {
                    return;
                }